### PR TITLE
Add e2e in Release Piepline

### PR DESCRIPTION
**Pull request by LianwMS**

### CODE REVIEW

File: .scripts/e2e/testcases.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    runTask(['azdev', 'extension', 'add', rp]);
+
+    logStep ("Azdev extension list");
+    runTask(['azdev', 'extension', 'list', '--query', '[?install==\'Installed\'].{Name:name,Path:path}', '-o', 'table']);
+
+    logStep ("Azdev linter: %s" % rp);
+    runTask(['azdev', 'linter', rp]);
+
+    logStep ("Azdev test: %s" % rp);
+    runTask(['azdev', 'test', rp, '--live']);
```
Comment (qiaozha) [inline .scripts/e2e/testcases.py:45]: If this RP has not been merged into the azure-cli-extensions repo. you will need --discover when run this azdev test.

File: azure-pipelines/release.yml [hunk 2]
```diff
@@ -57,6 +57,66 @@ stages:
...
+          echo -e "\e[32m[$(date -u)] LOG: Azdev setup"
+          azdev setup -r ../azure-cli-extensions
+        displayName: 'Prepare env'
+      - script: |
+          CURRENT_VERSION=$(node -p "require('./package.json').version")
+          echo -e "\e[32m[$(date -u)] LOG: CURRENT AZ VERSION: $CURRENT_VERSION"
+          echo -e "\e[32m[$(date -u)] LOG: Active venv"
+          . /opt/venv/bin/activate
+          echo -e "\e[32m[$(date -u)] LOG: Go to azure-cli folder"
+          cd ../azure-cli
```
Comment (qiaozha) [inline azure-pipelines/release.yml:111]: Here should be azure-cli-extensions folder right?

File: .scripts/e2e/testcases.py (hunk 1 above)
Comment (LianwMS) [reply .scripts/e2e/testcases.py:45]: Not understand, sync offline

File: azure-pipelines/release.yml (hunk 2 above)
Comment (LianwMS) [reply azure-pipelines/release.yml:111]: Not understand, sync offline

Comment (LianwMS) [reply azure-pipelines/release.yml:111]: add --discover

File: .scripts/e2e/testcases.py (hunk 1 above)
Comment (LianwMS) [reply .scripts/e2e/testcases.py]: add --discover

**Pull request merged**